google-cloud-spanner/acceptance/spanner_helper.rb

```diff
@@ -185,14 +185,35 @@ def items_ddl_statement
         ITEMS
       end
 
+      def numeric_pk_ddl_statement
```

Comment:
These added functions need to check if the emulator is enabled before referencing `NUMERIC`, similar to `stuffs_ddl_statement`. Currently, they are causing failures against the emulator.

Reply:
@larkee  updated.